gaia-button.js

```diff
@@ -52,6 +70,51 @@ module.exports = component.register('gaia-button', {
           this.removeAttr('disabled');
         }
       }
+    },
+
+    keyboardEnabled: {
+      get: function() { return this.hasAttribute('keyboardEnabled'); },
+      set: function(value) {
+        value = !!(value === '' || value);
+
+        if (value === true) {
+          this.setAttr('keyboardEnabled', '');
+          this.addEventListener('keydown', this._handleKeyEvent.bind(this));
+          this.addEventListener('keyup', this._handleKeyEvent.bind(this));
+        } else {
+          this.removeAttr('keyboardEnabled');
+          this.removeEventListener('keydown', this._handleKeyEvent.bind(this));
+          this.removeEventListener('keyup', this._handleKeyEvent.bind(this));
+        }
+      }
+    },
+
+    toggled: {
```

Comment:
It is for displaying as a toggle button. TV has it. gaia-button should also need it. How do you feel?